### PR TITLE
Use kube API proxy

### PR DESCRIPTION
**Pull request by mohabusama**
+ Add option to use API proxy (for offloading auth and dropping the use of serviceaccount)
+ Fix appdynamics template
+ Skip **pause** containers

### CODE REVIEW

File: k8s_log_watcher/main.py [hunk 1]
```diff
@@ -234,9 +243,17 @@ def main():
     containers_path = os.environ.get('WTACHER_CONTAINERS_PATH', args.containers_path)
     dest_path = os.environ.get('WTACHER_DEST_PATH', args.dest_path)
 
+    kube_url = os.environ.get('WTACHER_KUBE_URL', args.kube_url)
```
Comment (mikkeloscar) [inline k8s_log_watcher/main.py]: should this be `WATCHER_KUBE_URL`?

Comment (mohabusama) [inline k8s_log_watcher/main.py]: Yea. A typo that propagated -.-

**Comment by mikkeloscar** [comment]
I was confused by the name `kube proxy` because I was thinking of the [kube-proxy](http://kubernetes.io/docs/admin/kube-proxy/).

Essentially the kube_url is just the URL to the apiserver/master so I would remove any mention of *kube proxy* as this is just a detail of how you deploy it.

**Comment by mohabusama** [comment]
Right. Will change it. Just to clarify, it is not direct URL to apiserver, it is rather URL to some "API proxy service" which handles the auth part and expected to run on the node.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**